repository.yaml

```diff
@@ -1,2 +1,2 @@
+digital-service: API Platform
```

Comment:
Where is this from? I would've thought that we'd want to keep MTD

Reply:
dunno but it's in all the others